python/py-aiosmtpd/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           python 1.0
 name                py-aiosmtpd
 version             1.4.2
```

Comment:
Thank you; could you please bump the revision as well?

Reply:
Since we're not changing the files that are installed, we shouldn't require a revision bump here. More info in [the docs](https://guide.macports.org/#reference.keywords).

Reply:
> could you please bump the revision as well?

This change doesn’t affect any code, so to the best of my knowledge a revbump is unnecessary.

Reply:
> Since we're not changing the files that are installed, we shouldn't require a revision bump here. More info in the docs.

I stand corrected, then! I'll merge this!